### DISCUSSION TITLE
R wrapper of Quibller

### DISCUSSION THREAD

**Discussion by gfialk** [Technion-Kishony-lab/quibbler#573] [Ideas]
Thanks for this great application!
Is there any wrapper of Quibbler for R programmers (in particular one that supports ggplot)?
That would be much appreciated...

**Comment by rkishony** [comment]
Quibbler for R would be a truly worthy undertaking indeed! 

It is something we ultimately want to do, but is a major undertaking and will require much work and collective effort. 

let's keep eye on how this is voted for.

and if anyone is interested/capable of helping to develop an "R-Quibbler" let me know
